opte-api/src/ip.rs

```diff
@@ -814,6 +855,26 @@ impl FromStr for IpCidr {
     }
 }
 
+#[cfg(feature = "ipnetwork")]
```

Comment:
I don't see an `"ipnetwork"` feature in `opte-api/Cargo.lock`. Did you mean to add one, or make this `feature = "std"`? This shows up a few other places too.

Reply:
Optional deps in `Cargo.toml` implicitly define a corresponding feature.

Reply:
I did not know that! Cool!